Google.Api.Generator/CodeGenerator.cs

```diff
@@ -38,7 +38,7 @@ internal static class CodeGenerator
         /// <summary>
         /// Extension registry with everything we need in.
         /// </summary>
-        private static readonly ExtensionRegistry s_registry = new ExtensionRegistry
+        private static readonly ExtensionRegistry Registry = new ExtensionRegistry
```

Comment:
Not sure whether the changes in the first commit are really worth it, to be honest - we still have other static readonly fields names `s_xyz`. Don't really mind either way.

Reply:
It was mostly to standardize it on this particular file, where we already had a mix. I'll leave the changes in and we can change the others progressively, or not.